lib/src/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="custom-breadcrumb" [ngClass]="className" [attr.id]="id">
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <ng-container *ngFor="let breadcrumb of list; index as i">
+                <li
+                    class="breadcrumb-item"
+                    [ngClass]="{ active: i === list.length - 1 }"
```

Comment:
since there is only one condition you could just do `[class.active]="i === list.length-1"`